src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/MissingOverride/context/case2/expected.txt

```diff
@@ -0,0 +1 @@
+Test.java:6:5: Must include @java.lang.Override annotation when {@inheritDoc} Javadoc tag exists.
```

Comment:
can you explain why we do not have violation on line 12 ? where we add javadoc (looks like simple case )

Reply:
@romani 

as mentioned in https://github.com/checkstyle/patch-filters/pull/133#discussion_r458848465, 
MissingOverride check is not Comments aware, so although we add comments, but it can not be captured by context strategy.

```
+    /** {@inheritDoc} */
     private void test3() {}  // violation without filter
 
```

```
|       `--RCURLY -> } [9:25]
    |--METHOD_DEF -> METHOD_DEF [12:4]
    |   |--MODIFIERS -> MODIFIERS [12:4]
    |   |   `--LITERAL_PRIVATE -> private [12:4]
    |   |--TYPE -> TYPE [12:12]
    |   |   `--LITERAL_VOID -> void [12:12]
    |   |--IDENT -> test3 [12:17]
    |   |--LPAREN -> ( [12:22]
    |   |--PARAMETERS -> PARAMETERS [12:23]
    |   |--RPAREN -> ) [12:23]
    |   `--SLIST -> { [12:25]
    |       `--RCURLY -> } [12:26]
    `--RCURLY -> } [14:0]
```